File: core/semantic/incompleteness_preservation_engine.py
from __future__ import annotations

from typing import Any, Dict

from core.evidence.incompleteness_engine import preserve_incompleteness


def preserve_semantic_incompleteness(bundle: Dict[str, Any]) -> Dict[str, Any]:
    inc = preserve_incompleteness(bundle)
    bundle["incompleteness"] = inc
    return bundle

File: src/semantic/incompletenessPreservationEngine.ts
/**
 * Converted from Python: core/semantic/incompleteness_preservation_engine.py
 * @generated — WebWeaveX python→javascript library port
 */

import * as py from "../runtime/pyCompat.js";
import { preserveIncompleteness } from "../evidence/incompletenessEngine.js";

export function preserveSemanticIncompleteness(bundle: any): any {
  var inc: any = preserveIncompleteness(bundle);
  py.setItem(bundle, "incompleteness", inc);
  return bundle;
}
export { preserveIncompleteness };
